dogfood/main.tf

```
@@ -172,10 +171,16 @@ resource "coder_agent" "dev" {
   startup_script         = <<-EOT
     set -eux -o pipefail
 
+    # change to home
+    cd /home/coder
```

Suggestion: This feels more future-proof.

```suggestion
    cd "$HOME"
```

---

It is run as the same user as the agent is launched as, in the case of this template, it's the `coder` user:

```
❯ ps aux|grep coder\ agent\$
coder          1  0.0  0.0 775636 49712 ?        Ssl  13:11   0:00 ./coder agent
```